src/EventListener/BlameListener.php

```diff
@@ -48,10 +48,13 @@ public function onKernelRequest(RequestEvent $event)
         }
     }
 
+    /**
+     * @return string[]
+     */
     public static function getSubscribedEvents()
```

Comment:
Thanks for the PR. Is there a reason to not use native return type? **: array**

Reply:
yes, because the class is not `final`, so adding a return type would be BC break (someone may have extended the class).

Reply:
ok, thanks for explanation